wpilib/src/pneumatics.rs

```diff
@@ -25,8 +25,12 @@ impl PneumaticsControlModule {
     // 0 is guaranteed to be valid.
     const DEFAULT: Self = PneumaticsControlModule(0);
 
+    pub fn new() -> Self {
+        Self::default()
+    }
+
     /// Creates a PCM object for a given ID.
-    pub fn new(module_id: i32) -> Option<Self> {
+    pub fn with_module(module_id: i32) -> Option<Self> {
```

Comment:
Hm. Is this a good idea? In the most common use case, teams would never have to construct a `PneumaticsControlModule` themselves.

Reply:
I was thinking along the lines of having `::new() == ::default`, like `Compressor`, another thing most teams would never make. (Though even fewer would need `PneumaticsControlModule`.)

The way I see it, there's the most common case, no PCM. Then there are two other uses, each of which is super uncommon to the point I can't guess which is more uncommon. One is to clear 
 / check faults, the other is to use multiple PCMs. With that in mind, `::new()` would probably see about the same use as `::with_module(x)` x != 0.

I think there is one other benefit to this way, though. If somebody doesn't know about `Default` or forgets to check it, then they might have to look up the right ID and end up with `PneumaticsControlModule::with_module(0)` instead of `::new()`.

At the end of the day though, whether there is a `::new()` alias for `::default` or not doesn't affect me at all, so it's not a huge deal to me.

Reply:
Hmm. Well, either way, `with_module` isn't an appropriate name here. This _is_ the module.

Reply:
Good point. What about `PneumaticsControlModule::with_id(module_id: i32)`?

Reply:
If we're going to do this, sounds good. Probably also be worth renaming the `number` method to match.